google/cloud/storage/client.h

```diff
@@ -131,7 +131,11 @@ class Client {
    *     Valid types for this operation include `MaxResults`, `Prefix`,
    *     `UserProject`, and `Projection`.
    *
-   * @throw std::runtime_error if the operation fails.
+   * @throw std::runtime_error if there is a permanent failure, or if there more
```

Comment:
"if there more" --> "if there were more"

...here and elsewhere.

Reply:
Done.